src/Microsoft.ML.Dnn/LearningRateScheduler.cs

```diff
@@ -192,4 +192,52 @@ internal override float GetLearningRate(TrainState trainstate)
         }
 
     }
+    /// <summary>
+    /// This class implements polynomial Learning rate decay.
+    /// Implemented from the tensorflow documentation.
+    /// Source: https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/polynomial_decay
+    /// Default values and implementation of learning rate is from Tensorflow Slim model tests.
+    /// Source : https://github.com/tensorflow/models/blob/master/research/slim/train_image_classifier.py
+    /// </summary>
+    public sealed class PolynomialLRDecay : LearningRateScheduler
+    {
+        public float LearningRate;
```

Comment:
These should all be `readonly`. You don't want people changing these values once the object is created.